### PR TITLE
fix(nuxt): include tag attrs for non self-closing tags in tree-shake regex

### PR DESCRIPTION
**Pull request by huang-julien**
### 🔗 Linked issue
resolve https://github.com/nuxt/framework/issues/6673

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Hi, this PR aims to add missing attributes when the tag is not a self-closing tag by fixing the regex. Previously, attributes were missing on non self-closing tags.

Test fixtures of this PR can be considered as included by PR https://github.com/nuxt/framework/pull/6584 ?

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/nuxt/src/components/tree-shake.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export const TreeShakeTemplatePlugin = createUnplugin((options: TreeShakeTemplat
       const s = new MagicString(code)
 
       // Do not render client-only slots on SSR, but preserve attributes
-      s.replace(COMPONENTS_RE, r => r.replace(/<([^ >]*)[ >][\s\S]*$/, '<$1 />'))
+      s.replace(COMPONENTS_RE, r => r.replace(/<([^>]*)(>|[/>])[\s\S]*$/, '<$1 />'))
```
Comment (danielroe) [inline packages/nuxt/src/components/tree-shake.ts:38]:
```suggestion
      s.replace(COMPONENTS_RE, r => r.replace(/<([^>]*[^\/])\/?>[\s\S]*$/, '<$1 />'))
```

Comment (huang-julien) [reply packages/nuxt/src/components/tree-shake.ts:38]: yes thank you ! `[^\/]` is escaping necessary ? my IDE warns about an unecessary escape

Comment (danielroe) [reply packages/nuxt/src/components/tree-shake.ts]: Ah, no it's not!

**Review by danielroe — APPROVED** [review]

**Comment by pi0** [comment]
Thanks @huang-julien @danielroe <3

**Pull request merged**